recipes/aif360/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "aif360" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c4b44742984c741e6dab1770afc65480b1fcd8e04d33549c346b03674b6b8dd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5, <3.8
```

Comment:
I can see upstream that the README says py3.8 isn't supported, but the `setup.py` doesn't really make any restrictions on max version, so you might consider dropping the `<3.8`.